src/ClassApplication/Controllers/ValuesController.cs

```diff
@@ -25,13 +25,47 @@ public ClassController(IHttpClientFactory factory, DatabaseService databaseServi
         /// Visit the microservice's endpoint and append /swagger
         /// to see your docs in action
         /// </summary>
-        /// <param name="id">Some sort of Id</param>
+        /// <param name="">Some sort of Id</param>
         /// <returns>An array containing a value determined by the parameter</returns>
         [HttpGet("{id}")]
         public async Task<ActionResult<string>> GetValues(string id)
         {
             await this.classesContainer.CreateItemAsync<dynamic>(new Object() { });
             return ("Created an object");
         }
+        
+        ///<summary>
+        ///This call is used to add a User to the Classroom
+        /// using the User ID generated by the Users Classroom
+        /// </summary>
+        /// <param name="classID">GUID ID for the classroom to add user too</param>
+        /// <param name="userID"> GUID ID of the user </param>
+        [HttpPut("{classID}/Users/{userID}")]
+        public async Task<ActionResult<string>> AddUserToClass(string classID, string userID){
+            throw new NotImplementedException();
+        }
+
+        ///<summary>
+        ///This call is used to remove a User from the Classroom
+        /// using the User ID generated by the Users Classroom
+        /// </summary>
+        /// <param name="classID">GUID of the classroom</param>
+        /// <param name="userID"> GUID ID of the user </param>
+        [HttpDelete("{classID}/Users/{userID}")]
+        public async Task<ActionResult<string>> RemoveUserFromClass(string classID, string userID){
+            throw new NotImplementedException();
+        }
+
+
+        ///<summary>
+        ///This call is used to search for a User by name
+        /// using the name associated with the user
+        ///</summary>
+        /// <param name="classID"> GUID of the classroom</param>
+        /// <param name="userName"> Name of the User</param>
+        [HttpGet("{classID}/Users/{userName}")]
+        public async Task<ActionResult<string>> QueryUser(string classID, string userName){

```

Comment:
nit: `QueryUser` may be too vague. It looks like you want to query for a user in a classroom? I would suggest `QueryUserInClassroom`